modules/vcf/utils.nf

```diff
@@ -18,6 +18,27 @@ def getHpoIds(samples) {
   samples.collectMany { sample -> sample.hpo_ids }.unique()
 }
 
+def getProbandHpoIds(samples) {
+  samples.findAll{ sample -> sample.proband }.collectMany { sample -> sample.hpo_ids }.unique()
+}
+
+def areProbandHpoIdsIndentical(samples) {
+  def hpo_ids=[]
+  def isIdentical = true
+  samples.findAll{ sample -> sample.proband }.each{ sample ->
+    {
+      if(hpo_ids.isEmpty() && !sample.hpo_ids.isEmpty()){
+        hpo_ids = sample.hpo_ids
+      }else{
+        if(sample.hpo_ids as Set != hpo_ids as Set){
```

Comment:
does this work? in Java it would check if both Objects are located at the same memory address instead of checking for object equality.

Reply:
I tested it by modifying our tests to have two probands with different hpo sets, and it worked